src/pmi/pmi.cs

```diff
@@ -1313,7 +1316,7 @@ private static void EnsureTimerCallbackIsJitted()
             callback: new TimerCallback(DummyTimerCallback),
             state: null,
             dueTime: 10,
-            period: Timeout.Infinite);
+            period: 1000);
```

Comment:
Wouldn't it be better to start it disabled, let the assignment happen, then change it, e.g.,:
```
timer = new Timer(
            callback: new TimerCallback(DummyTimerCallback),
            state: null,
            dueTime: Timeout.Infinite,
            period: Timeout.Infinite);
timer.Change(10, Timeout.Infinite);
```

then, no change needed in `DummyTimerCallback`?

Reply:
Thanks. That seems simpler.